ext/dom/node.c

```diff
@@ -603,11 +603,11 @@ int dom_node_prefix_read(dom_object *obj, zval *retval)
 
 int dom_node_prefix_write(dom_object *obj, zval *newval)
 {
-	zend_string *str;
+	zend_string *prefix;
 	xmlNode *nodep, *nsnode = NULL;
 	xmlNsPtr ns = NULL, curns;
 	char *strURI;
-	char *prefix;
+	char *prefix_char;
```

Comment:
This makes it sound like prefix is a single character...

Reply:
Yeah now that you mention it, any idea of a better name? Or should I rename this to prefix, and the zend_string to prefix_str?

Reply:
Sounds reasonable